xray/services/scan.go

```diff
@@ -91,7 +91,11 @@ func (ss *ScanService) ScanGraph(scanParams XrayGraphScanParams) (string, error)
 	}
 
 	if err = errorutils.CheckResponseStatus(resp, http.StatusOK, http.StatusCreated); err != nil {
-		return "", errorutils.CheckError(errorutils.GenerateResponseError(resp.Status, clientutils.IndentJson(body)))
+		scanError := ScanError{}
```

Comment:
scanError := ScanError{}
-->
scanErrorJson := ScanErrorJson{}

(`ScanError` looks like a type of `error`)